config/config_test.go

```diff
@@ -10,6 +10,39 @@ import (
 	"go.viam.com/utils"
 )
 
+const CloudStoryEnabled = "cloud_story_enabled"
+
+func configWithoutRequiredFieldsTestHelper(
```

Comment:
This was pulled out from an existing test to use to test with & without the feature flag